radio/src/lua_api.cpp

```diff
@@ -104,12 +104,72 @@ void hook(lua_State* L, lua_Debug *ar)
   }
 }
 
+#if defined(PCBTARANIS) && defined(REV9E)
+  #define RADIO "taranisx9e"
```

Comment:
I am not happy whit this part of the code also, but I needed to hack up something. Do you have an example of the right solution?